src/index.ts

```diff
@@ -83,6 +90,35 @@ async function addApp(name: string | undefined, options: { folderOnly?: boolean,
     }
 }
 
+// Add an app or folder to sparse-checkout
+async function removeApp(name: string | undefined, options: { folderOnly?: boolean, apps?: string }) {
+    if (!name && !options.apps) {
```

Comment:
Do we need to add validation for either folderOnly or apps , and mandatory name?